### PR TITLE
Proposed updates to kromer plots

### PR DESCRIPTION
**Pull request by jamesgillanders**
In this PR, I am proposing new updates to tardis_kromer_plot.py, which now will only plot the elements in the model in the colormap. This leads to a much less cluttered colormap. Additionally, elements up to and including U have been added to enable trans Fe-group elements to be included. A new flag has also been introduced to enable the user to limit the number of elements to be included in the colormap. This is not necessary in most cases, but useful for models where many elements have been included, but most have negligible effects on the spectrum.

### CODE REVIEW

**Comment by jamesgillanders** [comment]
need to add a comment that if using nelements flag, it is possible for the user to end up with features missing from the kromer plot

**Comment by jamesgillanders** [comment]
Need to modify the code to take the list of elements to use in the colour bar from descending order of frequency within the specified wavelength range, as opposed to the current method, which computes this across the entire wavelength range of the model

**Comment by jamesgillanders** [comment]
this code is ready for final review and merge. All suggested improvements and edits have been implemented

**Comment by jamesgillanders** [comment]
> need to add a comment that if using nelements flag, it is possible for the user to end up with features missing from the kromer plot

have added this in the docs

**Comment by jamesgillanders** [comment]
> Need to modify the code to take the list of elements to use in the colour bar from descending order of frequency within the specified wavelength range, as opposed to the current method, which computes this across the entire wavelength range of the model

have added this functionality

**Comment by jamesgillanders** [comment]
> Doesn't conform to PEP8 but otherwise looks good to me

ran Black on the code and it should now conform to PEP8 standards

File: tardis_kromer_plot.py [hunk 1]
```diff
@@ -6,28 +6,121 @@
...
-            14, 'p': 15, 's': 16, 'cl': 17, 'ar': 18, 'k': 19,    'ca': 20,
-            'sc': 21, 'ti': 22, 'v': 23, 'cr': 24, 'mn': 25, 'fe': 26, 'co':
-            27, 'ni': 28, 'cu': 29, 'zn': 30, 'ga': 31, 'ge': 32, 'as': 33,
-            'se': 34, 'br': 35, 'kr': 36, 'rb': 37, 'sr': 38, 'y': 39,  'zr':
-            40, 'nb': 41, 'mo': 42, 'tc': 43, 'ru': 44, 'rh': 45, 'pd': 46,
-            'ag': 47, 'cd': 48}
+"""the elements dictionary looks like this because this is how black linter
+formatted it - until this info is moved and stored in a separate file, it will
+remain here like this"""
+elements = {
```
Comment (jaladh-singhal) [inline tardis_kromer_plot.py:29]: For now, we can put this in a csv file in same directory in this repository, and read it from there - preferably by pandas!

Comment (jamesgillanders) [reply tardis_kromer_plot.py]: I have added this functionality now - see PR #29

**Pull request merged**